### DISCUSSION TITLE
How to set bitmap with scrolling text and a MQTT question

### DISCUSSION THREAD

**Discussion by Rushmed** [pixelit-project/PixelIt#131] [Q&A] [answered]
Hi.

First thanks for this awsome project!

I am using PixelIt wit ioBroker an the ioBroker Adapter. MQTT is also available but I dont really like that it updates mqtt.0.PixelIt.1.luxsensor every two seconds although it is not even available.

How to set the display to show a .bmp (8x8) on the left and on the rest a scrolling text with adjustable scrolling speed usinf the adapter?

I am using an AM312 PIR sensor. Is there a way to add the functionality of triggering on high level to "on" and with adjustable timeout to "off". Unfortunately teh sensor (button) is not listed in the adapter.+

**Comment by o0shojo0o** [comment]
for the display of a screen via the adapter i can give you this link :
https://docs.bastelbunker.de/ioBroker_pixelit/options.html#datenpunkte

**Comment by Rushmed** [comment]
OK, and how to set a bitmap ID from galery using json string?

**Comment by o0shojo0o** [comment, marked as answer]
This is not possible because the PixelIt has no online connections except for the WebUI. 
When you send a JSON to the PixelIt you have to send the byte array.
